logger/logger.go

```diff
@@ -0,0 +1,23 @@
+// Package config defines the Cloudinary configuration.
+package logger
+
+import (
+	"log"
+)
+
+// Logger cloudinary logger
+type LoggerFunction func(...interface{})
+
+type Logger struct {
```

Comment:
This interface seems to be very generic, which is nice, but it lacks support for setting logging level. like, I want to use` a.Logger.DebugLog()` without printing anything until unless logger level is set to debug. currently i need to configure each logger separately.

Reply:
And you could do it by overriding the DebugLog function - I gave the example of it in `example/example.go`.
The idea is that to configure the logger you should just override a function, and the function might be dummy.